src/polyfills/index.js

```diff
@@ -43,7 +43,7 @@ if (!IS_ESM) {
   installArrayIncludes(self);
   installMapSet(self);
   installWeakMapSet(self);
-  installSetAdd(self);
+  installSet(self);
```

Comment:
Strange that this list isn't kept in sync with https://github.com/ampproject/amphtml/blob/33dd4282678b748014ceea1424bc7134c482c93e/src/friendly-iframe-embed.js#L698-L701.